common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -1205,6 +1205,7 @@
   "feature.wynntils.territoryAttackTimerOverlay.overlay.territoryAttackTimer.textShadow.description": "What should the text shadow look like?",
   "feature.wynntils.territoryAttackTimerOverlay.overlay.territoryAttackTimer.textShadow.name": "Text Shadow",
   "feature.wynntils.territoryDefenseMessage.description": "Sends the territory defense message when you attack a territory.",
+  "feature.wynntils.territoryDefenseMessage.message": "%s defense is %s",
```

Comment:
Might be wrong but don't we get the defense from this chat message in the attack timer model? Probably not a good idea to make it translatable if so

Reply:
That's right, that's almost the whole point of the message.

Reply:
Can we not get it from the advancement? Or does that not update frequently enough

Reply:
The one in the attack menu is basically resource tick accurate, the advancement one updates like every 10 mins or so